docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`6636` Aave v3 positions and liabilities will now be properly shown in the dashboard and the defi sections.
```

Comment:
I would omit the defi section since it is only for ethereum but let @LefterisJP say here

Reply:
yes + 1. I agree. Remove defi section.